### PR TITLE
[SD-3149] Allow multiple index to be created per collection.

### PR DESCRIPTION
**Pull request by mdhaman**

### CODE REVIEW

File: server/apps/prepopulate/app_initialization_test.py [hunk 1]
```diff
@@ -37,10 +38,20 @@ def data_scaffolding_test(self):
...
+
+    def test_app_initialization_index_creation(self):
+        result = app.data.mongo.pymongo(resource='users').db['users'].index_information()
+        self.assertEqual(result, {})
+        command = AppInitializeWithDataCommand()
+        result = command.run()
+        self.assertEqual(result, 0)
+        result = app.data.mongo.pymongo(resource='users').db['users'].index_information()
+        self.assertTrue('username_1' in result)
+        self.assertTrue('first_name_1_last_name-1')
```
Comment (mugurrus) [inline server/apps/prepopulate/app_initialization_test.py]: Missing "in result"

**Comment by ioanpocol** [comment]
:+1:

**Comment by mugurrus** [comment]
:+1:

File: server/apps/prepopulate/app_initialization_test.py (hunk 1 above)
Comment (mdhaman) [inline server/apps/prepopulate/app_initialization_test.py]: correct.

**Comment by sivakuna-aap** [comment]
:+1: Can we add some doc string for each tuple value in __entities__?

**Pull request merged**